src/app/pages/details/customization-form/customization-form.js

```diff
@@ -0,0 +1,209 @@
+import React, {useState} from 'react';
+import {RawHTML} from '~/components/jsx-helpers/jsx-helpers.jsx';
+import FormInput from '~/components/form-input/form-input.jsx';
+import ModuleList from './module-list';
+import {useUserStatus} from '../common/hooks';
+import showDialog, {hideDialog} from '../common/show-dialog';
+import {cmsPost} from '~/models/cmsFetch';
+import './customization-form.css';
+
+// This stuff will come from the CMS
+const description = 'Please select the modules (up to 10) that you want to customize with Google Docs.';
+const disclaimer = `
+    <h2>Disclaimer</h2>
+    <p>The following features and functionality are not available to
+    teachers and students using Google Docs customized content:
+    </p>
+    <ul>
+        <li>
+            <b>Errata updates.</b> OpenStax webview is updated at least twice
+            yearly. Customized Google Docs will not receive these content updates.
+        </li>
+        <li>
+            <b>Access to study tools.</b> OpenStax webview has in-book search,
+            highlighting, study guides, and more available for free. This
+            functionality will not be available in Google Docs versions.
+        </li>
+        <li>
+            <b>Formatting.</b> Print books and webview have a specific design and
+            structure format developed for those platforms. These functionalities
+            are not available in the Google Docs versions.
+        </li>
+    </ul>
+`;
+const thankYou = 'Thank you &mdash; your request has been sent to the OpenStax team.';
+const nextSteps = `
+    <h2>Next Steps</h2>
+    <ol>
+        <li>Within two business days, you will receive an email for each module
+        that you have requested access to customize.</li>
+        <li>The link provided in the email will be y our own copy of the Google
+        Doc that OpenStax has generated for you.</li>
+        <li>Once you have accessed the document you can make the changes you
+        desire and share with your students. We recommend using the "Publish to
+        the Web" functionality under the File menu for sharing with students.</li>
+    </ol>
+`;
+
+function Feedback({data}) {
+    return (
```

Comment:
Instead of storing these as strings and rendering them each time, you can pre-render as jsx and store that, then insert it when needed.   It's a bit more efficient and you also get the editor support for JSX that way.

For instance this sort of thing works well:

```
const disclaimer = (
     <h2>Disclaimer</h2>
     <p>The following features and functionality are not available to
     teachers and students using Google Docs customized content:
     </p>
   ...
)
```

And then elsewhere:
```
   return (
      <div> message: {disclaimer} </div>
   )

```

On a side note, Codecov is making this hard to read 😡

Reply:
I'm anticipating them ultimately coming from the CMS.